### PR TITLE
new sorting method for treeviews, when TreeProvider is not used like …

### LINKED ISSUE #71: protocols are not sorted by name
and none of the columns can be sorted:
Screenshot_20200326_201412

### PR DESCRIPTION
**Pull request by pconesa**
…"Search protocol" dialog.

new method used, and sorting enabled in "Search protocol" dialog.

closes #71

### CODE REVIEW

**Review by fonsecareyna82 — APPROVED** [review]

File: pyworkflow/gui/tree.py [hunk 1]
```diff
@@ -629,4 +629,25 @@ def getText(self, obj):
         return obj.pluginName + '-' + obj.templateName
 
     def getValues(self, obj):
-        return (obj.description, )
\ No newline at end of file
+        return (obj.description, )
+
+# From: https://stackoverflow.com/questions/1966929/tk-treeview-column-sort
+def treeview_sort_column(tv, col, reverse, casting=str):
```
Comment (delarosatrevin) [inline pyworkflow/gui/tree.py:635]: We use camelCase convention for methods, please keep it consistent

Comment (delarosatrevin) [reply pyworkflow/gui/tree.py:635]: Moreover, why this method is not implemented as part of the class pwgui.tree.Tree (that is a subclass of Tkinter.Treeview) and it is more OO?

**Review by delarosatrevin — CHANGES_REQUESTED** [review]
Thanks, @pconesa.
I have proposed some changes.

File: pyworkflow/gui/tree.py (hunk 1 above)
Comment (pconesa) [reply pyworkflow/gui/tree.py:635]: Good points, I missed boths. Doing them. Thanks

**Review by delarosatrevin — APPROVED** [review]
Nice @pconesa!

**Comment by pconesa** [comment]
Closes  #134

**Pull request merged**